wc_langPacks/wc_langPack_EN.js

```diff
@@ -39,6 +40,17 @@ wc_addLanguagePack({
       'ELEVEN',
       'TWELVE'
     ];
+	// m=35;
+	if (((m + 1) % 5) == 0) {
+		ret += 'NEARLY ';
+		m += 1;
+	}
+	
+	if (((m + 2) % 5) == 0) {
+		ret += 'NEARLY ';
+		m += 2;
+	}
+	
```

Comment:
This can be written the following way (review for me, I'll change in an additional Commit).
```suggestion
	// 5+0, 5+1, 5+2 => not nearly
	// 5+3, 5+4 => nearly
	if (m  % 5 >= 3) {
		ret += 'NEARLY ';
		m += 2; // Always adding 2 works, because the switch rounds (down) to 5 minute intervals
	}
	
```

Reply:
Hi,

Yes, this is much neater! I've run the syntax through a test script for for m=0 to 59 and added it to my own build. Looks great, thank you!

Re the settings option vs extra language pack question, my feeling is that a settings option would be better, but my German (etc) isn't good enough to know if it's as easy to add 'nearly' in other languages.